frontend/src/metabase/admin/datamodel/components/database/ColumnItem/ColumnItem.jsx

```diff
@@ -59,7 +59,7 @@ class Column extends Component {
               <ColumnItemInput
                 variant="primary"
                 style={{ minWidth: 420 }}
-                className="AdminInput TableEditor-field-name float-left inline-block rounded text-bold"
+                className="TableEditor-field-name float-left inline-block rounded text-bold"
```

Comment:
Looks like we can remove `TableEditor-field-name rounded text-bold` here and below